governance/votes/Executive vote - April 16, 2021.md

```diff
@@ -0,0 +1,91 @@
+* The Liquidations 2.0 Contracts will be authorized.
+* LINK-A will be moved to the Liquidations 2.0 Framework.
+* The Emergency Shutdown Threshold will be increased from 50,000 MKR to 75,000 MKR.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Authorize Liquidations 2.0 Contracts
+
+Given the passing of this [executive](https://vote.makerdao.com/executive/6058cea1efe023001bfeb313?network=mainnet#proposal-detail) the following contracts will be authorized to implement Liquidations 2.0:
+* Add contract "MCD_DOG" - Main contract for Liquidations 2.0.
+* Replace contract "MCD_END" - Contract for executing a shutdown of the protocol.
+* Replace contract "MCD_ESM" - Emergency Shutdown Module contract that enables burning MKR to shut the protocol down.
+* Replace contract "ILK_REGISTRY" - Contract defining the different vault types within the protocol.
+* Add contract "CLIPPER_MOM" - Contract that enables a "circuit breaker" for Liquidations 2.0 if the price of a collateral drops too severely.
+* Add contract "MCD_CLIP_LINK_A" - Contract containing the LINK-A auction module (with the parameters listed below).
+* Add contract "MCD_CLIP_CALC_LINK_A" - Contract that calculates the auction price for LINK-A. 
+* Remove contract "MCD_FLIP_LINK_A" - The previous contract setting the LINK-A liquidation parameters.
+
+Contract addresses and updates may be found in the [Collateral Onboarding Updates Rocket.Chat Channel](https://chat.makerdao.com/channel/collateral-onboarding-updates-ext).
+
+Further details and rationale can be found in this [thread](https://forum.makerdao.com/t/mip45-liquidations-2-0-liq-2-0-liquidation-system-redesign/6352) or by browsing the [Liq2.0 Tag](https://forum.makerdao.com/tag/liq-20) in the forum.
+
+### Add LINK-A to Liquidations 2.0 Framework
+
+Given the successful outcome of this [poll](https://vote.makerdao.com/polling/QmPtNp9a?network=mainnet#poll-detail), LINK-A will be added to the Liquidations 2.0 Framework with the following parameters:
```

Comment:
'add if this proposal passes' when saying things like 'LINK-A will be added to the Liquidations 2.0 Framework with the following parameters'. We want to make it clear that it's not a foregone conclusion.